### ISSUE TITLE
Crash when clicking on own identity in contact list

### ISSUE THREAD

**Issue by audax** [Qabel/qabel-android#597]
Scenario: Installed the app, created 1 identity

The application crashed when selecting the identity in the contact list.

```
07-15 13:19:10.203 8395-8395/de.qabel.qabel.debug E/AndroidRuntime: FATAL EXCEPTION: main
                                                                    Process: de.qabel.qabel.debug, PID: 8395
                                                                    rx.exceptions.OnErrorNotImplementedException: List is empty.
                                                                        at rx.internal.util.InternalObservableUtils$ErrorNotImplementedAction.call(InternalObservableUtils.java:374)
                                                                        at rx.internal.util.InternalObservableUtils$ErrorNotImplementedAction.call(InternalObservableUtils.java:371)
                                                                        at rx.internal.util.ActionSubscriber.onError(ActionSubscriber.java:44)
                                                                        at rx.observers.SafeSubscriber._onError(SafeSubscriber.java:157)
                                                                        at rx.observers.SafeSubscriber.onError(SafeSubscriber.java:120)
                                                                        at rx.exceptions.Exceptions.throwOrReport(Exceptions.java:204)
                                                                        at rx.observers.SafeSubscriber.onNext(SafeSubscriber.java:144)
                                                                        at de.qabel.qabelbox.contacts.interactor.MainContactsUseCase$loadContact$1.invoke(MainContactsUseCase.kt:46)
                                                                        at de.qabel.qabelbox.contacts.interactor.MainContactsUseCase$loadContact$1.invoke(MainContactsUseCase.kt:22)
                                                                        at rx.lang.kotlin.ObservablesKt$sam$OnSubscribe$5dafc494.call(observables.kt)
                                                                        at rx.lang.kotlin.ObservablesKt$sam$OnSubscribe$5dafc494.call(observables.kt)
                                                                        at rx.Observable.subscribe(Observable.java:8553)
                                                                        at rx.Observable.subscribe(Observable.java:8520)
                                                                        at rx.Observable.subscribe(Observable.java:8343)
                                                                        at de.qabel.qabelbox.contacts.view.presenters.MainContactDetailsPresenter.refreshContact(MainContactDetailsPresenter.kt:17)
                                                                        at de.qabel.qabelbox.contacts.view.views.ContactDetailsFragment.onActivityCreated(ContactDetailsFragment.kt:53)
                                                                        at android.app.Fragment.performActivityCreated(Fragment.java:2228)
                                                                        at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:992)
                                                                        at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1148)
                                                                        at android.app.BackStackRecord.run(BackStackRecord.java:793)
                                                                        at android.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1535)
                                                                        at android.app.FragmentManagerImpl$1.run(FragmentManager.java:482)
                                                                        at android.os.Handler.handleCallback(Handler.java:739)
                                                                        at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                        at android.os.Looper.loop(Looper.java:148)
                                                                        at android.app.ActivityThread.main(ActivityThread.java:5417)
                                                                        at java.lang.reflect.Method.invoke(Native Method)
                                                                        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
                                                                        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
                                                                     Caused by: java.util.NoSuchElementException: List is empty.
                                                                        at kotlin.collections.CollectionsKt___CollectionsKt.first(_Collections.kt:180)
                                                                        at de.qabel.qabelbox.contacts.view.adapters.ContactDetailsAdapter.loadContact(ContactDetailsAdapter.kt:36)
                                                                        at de.qabel.qabelbox.contacts.view.views.ContactDetailsFragment.loadContact(ContactDetailsFragment.kt:88)
                                                                        at de.qabel.qabelbox.contacts.view.presenters.MainContactDetailsPresenter$refreshContact$1.call(MainContactDetailsPresenter.kt:17)
                                                                        at de.qabel.qabelbox.contacts.view.presenters.MainContactDetailsPresenter$refreshContact$1.call(MainContactDetailsPresenter.kt:9)
                                                                        at rx.internal.util.ActionSubscriber.onNext(ActionSubscriber.java:39)
                                                                        at rx.observers.SafeSubscriber.onNext(SafeSubscriber.java:139)
                                                                        at de.qabel.qabelbox.contacts.interactor.MainContactsUseCase$loadContact$1.invoke(MainContactsUseCase.kt:46) 
                                                                        at de.qabel.qabelbox.contacts.interactor.MainContactsUseCase$loadContact$1.invoke(MainContactsUseCase.kt:22) 
                                                                        at rx.lang.kotlin.ObservablesKt$sam$OnSubscribe$5dafc494.call(observables.kt) 
                                                                        at rx.lang.kotlin.ObservablesKt$sam$OnSubscribe$5dafc494.call(observables.kt) 
                                                                        at rx.Observable.subscribe(Observable.java:8553) 
                                                                        at rx.Observable.subscribe(Observable.java:8520) 
                                                                        at rx.Observable.subscribe(Observable.java:8343) 
                                                                        at de.qabel.qabelbox.contacts.view.presenters.MainContactDetailsPresenter.refreshContact(MainContactDetailsPresenter.kt:17) 
                                                                        at de.qabel.qabelbox.contacts.view.views.ContactDetailsFragment.onActivityCreated(ContactDetailsFragment.kt:53) 
                                                                        at android.app.Fragment.performActivityCreated(Fragment.java:2228) 
                                                                        at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:992) 
                                                                        at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1148) 
                                                                        at android.app.BackStackRecord.run(BackStackRecord.java:793) 
                                                                        at android.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1535) 
                                                                        at android.app.FragmentManagerImpl$1.run(FragmentManager.java:482) 
                                                                        at android.os.Handler.handleCallback(Handler.java:739) 
                                                                        at android.os.Handler.dispatchMessage(Handler.java:95) 
                                                                        at android.os.Looper.loop(Looper.java:148) 
                                                                        at android.app.ActivityThread.main(ActivityThread.java:5417) 
                                                                        at java.lang.reflect.Method.invoke(Native Method) 
                                                                        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
                                                                        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
```

**Comment by JoGir** [comment]
Is fixed in my current PR. Happens if a contact is not assigned to identities...

**Comment by JoGir** [comment]
fixed in #595
